### DISCUSSION TITLE
Attributes not showing after GraphML import

### DISCUSSION THREAD

**Discussion by yellek** [graphia-app/graphia#73] [General]
Is there something I have to do to get the attributes from a GraphML file to import?

The file below should have a name attribute attached to n0 but it doesn't.

`
<?xml version="1.0" encoding="UTF-8"?>
<graphml:graphml xmlns:graphml="http://graphml.graphdrawing.org/xmlns"  
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns
     http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">
<graphml:key id="arc-name" attr.type="string" attr.name="name"/>
  <graphml:graph id="G" edgedefault="directed">
    <graphml:node id="n0">
<graphml:data key="arc-name">Data Management</graphml:data>
    </graphml:node>
    <graphml:node id="n1"/>
    <graphml:node id="n2"/>
    <graphml:node id="n3"/>
    <graphml:node id="n4"/>
    <graphml:node id="n5"/>
    <graphml:node id="n6"/>
    <graphml:node id="n7"/>
    <graphml:node id="n8"/>
    <graphml:node id="n9"/>
    <graphml:node id="n10"/>
    <graphml:edge source="n0" target="n2"/>
    <graphml:edge source="n1" target="n2"/>
    <graphml:edge source="n2" target="n3"/>
    <graphml:edge source="n3" target="n5"/>
    <graphml:edge source="n3" target="n4"/>
    <graphml:edge source="n4" target="n6"/>
    <graphml:edge source="n6" target="n5"/>
    <graphml:edge source="n5" target="n7"/>
    <graphml:edge source="n6" target="n8"/>
    <graphml:edge source="n8" target="n7"/>
    <graphml:edge source="n8" target="n9"/>
    <graphml:edge source="n8" target="n10"/>
  </graphml:graph>
</graphml:graphml>
`

**Comment by timangus** [comment]
Your attribute declaration is missing a _for_ value:
`<graphml:key id="arc-name" for="node" attr.type="string" attr.name="name"/>`

**Reply by yellek** [reply]
Ah thanks - working now
